src/server.h

```diff
@@ -672,6 +673,9 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	std::unordered_map<std::string, ModMetadata *> m_mod_storages;
 	float m_mod_storage_save_timer;
 
+	// CSM flavour limits byteflag
+	u64 m_csm_flavour_limits = CSMFlavourLimits::CSM_FBL_NONE;
```

Comment:
For this to work, you need to `#include "networkprotocol.h` because that is where the `CSMFlavorLimits` is declared.